File: bindings/js/src/generators/AMFJSONLDGenerator.ts
/**
 * Created by antoniogarrote on 05/05/2017.
 */

import {BaseGenerator} from "./BaseGenerator";
import {Clojure} from "../Clojure";

/**
 * Generator that exports the AMF model as a JSON-LD document
 */
export class AMFJSONDGenerator extends BaseGenerator {
    protected generator(): any {
        return new Clojure.amf.__GT_APIModelGenerator();
    }
}

File: bindings/java/src/org/raml/amf/generators/AMFJSONLDGenerator.java
package org.raml.amf.generators;

/**
 * Created by antoniogarrote on 04/05/2017.
 */

/**
 * Generator that exports the AMF model as a JSON-LD document
 */
public class AMFJSONLDGenerator extends BaseGenerator {

    @Override
    protected String generatorConstructor() {
        return "->APIModelGenerator";
    }
}
